db/src/main/kotlin/edu/artic/db/models/ArticObject.kt

```diff
@@ -52,12 +51,9 @@ data class ArticObject(
         @Json(name = "reference_num") val referenceNum: String? = null,
         @Json(name = "audio_commentary") val audioCommentary: List<AudioCommentaryObject> = listOf(),
         @Json(name = "highlighted_object") val highlightedObject: String? = null,
-        @Deprecated("Please do not use the 'full' image, as it is too large to fit on screen. Migrate to 'image_url' or 'large_image_full_path' immediately.")
-        @Json(name = "full_image_full_path") val fullImageFullPath: String? = null,
 //        @Json(name = "audio") val audio: List<String?>, Removed for now until json gets fixed as sometimes returns string othertimes object
```

Comment:
This might be a good time to remove this too? We're retrieving audio properties in a different way (`audioCommentary`) now.

Reply:
@Cliabhach Removed it.